api/controllers/csv.js

```diff
@@ -52,6 +52,56 @@ exports.getReportPastWeekCSV = function (req, res, next) {
     });
 };
 
+exports.getProjectReportPastWeekCSV = function (req, res, next) {
+    Hours.find({
+        week: {
+            $gte: weeksSinceEpoch() - 1
+        }
+    }, function (err, docs) {
+        if (err) return next(err);
+
+        var ret = [];
+        var totalHours = {};
+
+        // first pass: get total hours for each user
+        // TODO make this a mongo static
+        _.forEach(docs, function (doc) {
+            _.forEach(doc.projects, function (project) {
+                if (totalHours.hasOwnProperty(doc.userId)) {
+                    totalHours[doc.userId] += project.hours;
+                } else {
+                    totalHours[doc.userId] = project.hours;
+                }
+            });
+        });
+
+        // second pass: record data and calculate percentages
+        _.forEach(docs, function (doc) {
+            _.forEach(doc.projects, function (project) {
+                var days = (doc.week * 7) + 4;
+                var epochTime = days * 8.64e7;
+                var date = new Date(epochTime).toISOString().slice(0, 10);
+                ret.push([
+                    project.name,
+                    doc.userName,
+                    date,
+                    project.hours,
+                    (project.hours/totalHours[doc.userId]).toFixed(2)
+                ]);
+            });
+        });
+
+        ret.sort((function(i) {
+            return function(a, b) {
+                return (a[i] === b[i] ? 0 : (a[i] < b[i] ? -1 : 1));
+            };
+        })(0));
+
+        res.set('Content-Disposition', 'attachment; filename="project.csv"');
```

Comment:
re: your comment on my PR, I'll align my routes to reflect yours.  Can you change the .csv export name on person to align with this convention though? (line 103)